### PR TITLE
Relative path resolves to project directory when use METEOR_LOCAL_DIR.

### PR DESCRIPTION
**Pull request by laosb**
Issue #7292 

Use `path.resolve` to resolve the relative path to project. May need to take this down to `History.md`.

### CODE REVIEW

File: tools/project-context.js [hunk 1]
```diff
@@ -89,9 +90,11 @@ _.extend(ProjectContext.prototype, {
     // writes to; used by `meteor test` so that you can test your
     // app in parallel to writing it, with an isolated database.
     // You can override the default .meteor/local by specifying
-    // METEOR_LOCAL_DIR.
+    // METEOR_LOCAL_DIR. You can use relative path if you want it
+    // relative to your project directory.
     self.projectLocalDir = process.env.METEOR_LOCAL_DIR ?
-      process.env.METEOR_LOCAL_DIR : (options.projectLocalDir ||
+      path.resolve(options.projectLocalDir, process.env.METEOR_LOCAL_DIR)
+      : (options.projectLocalDir ||
```
Comment (Slava) [inline tools/project-context.js]: for consistency with the rest of the codebase this should be `files.pathResolve(options.projectLocalDir, files.convertToStandardPath(process.env.METEOR_LOCAL_DIR))`

**Comment by Slava** [comment]
I can't comment if this is logically the correct fix, but the work with file-system paths should be very careful to work well on Windows. See https://github.com/meteor/meteor/tree/devel/tools/fs

**Comment by laosb** [comment]
Ah, did see the README but didn't notice that `path` also wrapped in `files`. Updated the code and fixed a former mistaken.

**Comment by laosb** [comment]
@Slava Tests passed!

**Comment by Slava** [comment]
I will leave the opportunity for @tmeasday to look at it since I have been detached from this codebase for awhile now.

**Pull request merged**